### PR TITLE
incorporating Optimize into the development process

### PR DESCRIPTION
**Pull request by christinaausley**
Minor adjustments to existing Cloud documentation to highlight Optimize getting started process.

### CODE REVIEW

**Comment by akeller** [comment]
Is this something we want to merge ASAP or will alerts come in 3.8?

File: docs/guides/improve-processes-with-optimize.md
Comment (HanselIdes) [inline docs/guides/improve-processes-with-optimize.md:85]: I should start enforcing common terminology. This should actually be a report with a number visualization rather than a single-value report

**Review by HanselIdes — DISMISSED** [review]
Thanks for working this in!
I have a few minor comments. Once they're addressed, feel free to merge it without requesting another review

File: docs/guides/improve-processes-with-optimize.md
Comment (christinaausley) [reply docs/guides/improve-processes-with-optimize.md:87]: Updated in latest commit.

File: docs/guides/improve-processes-with-optimize.md
Comment (christinaausley) [reply docs/guides/improve-processes-with-optimize.md:85]: Can add this to the [Components glossary](https://docs.google.com/document/d/1g1mLWK27HCiO_Jf-BT0lw65OBZHx-DzPk3xC8Bh-7Yw/edit) 👍

**Comment by christinaausley** [comment]
> Is this something we want to merge ASAP or will alerts come in 3.8?

Alerts are live in Cloud and came with 3.7 🥇

**Pull request merged**
